### PR TITLE
Adding workflow generate utils function

### PR DESCRIPTION
**Pull request by imrajdas**
Adding workflow generate utils function

### CODE REVIEW

File: pkg/chaos/workflow.go [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+type Template struct {
+	Name      string   `yaml:"name"`
+	Steps     [][]Step `yaml:"steps,omitempty"`
+	Input     Input    `yaml:"inputs,omitempty"`
+	Container struct {
+		Args    []string `yaml:"args"`
+		Command []string `yaml:"command"`
+		Image   string   `yaml:"image"`
+	} `yaml:"container,omitempty"`
+}
```
Comment (gdsoumya) [inline pkg/chaos/workflow.go]: Not all fields are considered specially metadata which might be needed in our case. ref: https://argoproj.github.io/argo/fields/#template

Comment (gdsoumya) [reply pkg/chaos/workflow.go]: Also take a look at the other structures and try making it match with the argo spec as workflows can be using some fields that aren't here and we would have to update these structs repeatedly.

**Review by gdsoumya — CHANGES_REQUESTED** [review]

**Review by gdsoumya — APPROVED** [review]
lgtm

**Review by arkajyotiMukherjee — APPROVED** [review]
lgtm

**Pull request merged**